.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+/ @libero/php-reviewers
```

Comment:
Not seen an example of apply it to the whole repo yet, but should work?

Reply:
Makes sense, let's try it